src/components/register/index.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { TextField, Box, makeStyles } from '@material-ui/core';
+
+import { BaseButton } from '../common';
+import Layout from '../layout';
+
+type DynamicFormInput = {
+    type: string,
+    label: string,
+    name?: string,
+    placeholder?: string,
+    required?: boolean,
+}
+
+type RegisterPageProps = {
+    favicon: string,
+    title: string,
+    dynamicFormInputs: DynamicFormInput[],
+};
+
+const useStyles = makeStyles(() => ({
+    form: {
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'space-evenly',
+        alignItems: 'center',
+        width: 320,
+        height: '70%',
+        overflow: 'auto'
+    },
+    input: {
+        width: 270,
+        '& label.Mui-focused': {
+            color: '#1976d2',
+        },
+        '& .MuiInput-underline:after': {
+            borderBottomColor: '#1976d2',
+        },
+    },
+}));
+
+// TODO: Add WebsiteInfo component at the top of this view
+export default function RegisterPage({ title, favicon, dynamicFormInputs }: RegisterPageProps) {
+    const classes = useStyles();
+
+    return (
+        <Layout>
+            <form className={classes.form}>
+                {
+                    dynamicFormInputs.map(props => {
+                        const { type, name, label, placeholder, required } = props;
+                        const isDateInput = type === 'date';
+                        const year = new Date().getFullYear();
+                        const defaultDate = `${year}-01-01`
+
+                        return (
+                            <Box>
+                                <TextField 
+                                    className={classes.input}
+                                    type={type}
+                                    label={label}
+                                    defaultValue={isDateInput ? defaultDate : ''}
+                                    name={name || ''}
```

Comment:
Appearance.
The WebsiteInfo component to show where this information is requested will be added later at the top of this component. (#8)
<img width="473" alt="스크린샷 2020-08-28 오전 12 38 41" src="https://user-images.githubusercontent.com/14817195/91463849-d6045c00-e8c6-11ea-9a8d-878e1e455fe7.png">